src/doc/rustc/src/codegen-options/index.md

```diff
@@ -244,18 +244,35 @@ flag for another way to specify the linker.
 
 ## linker-features
 
-This flag allows enabling or disabling specific features used during linking.
+The `-Clinker-features` flag allows enabling or disabling specific features used during linking.
 
-The flag accepts a comma-separated list of features, individually enabled (`+feature`) or disabled
-(`-feature`).
+These feature flags are a flexible extension mechanism that is complementary to linker flavors,
+designed to avoid the combinatorial explosion of having to create a new set of flavors for each
+linker feature we'd want to use.
 
-Currently, only one such feature is stable:
-- `lld`: toggles the usage of the lld linker, either the system-installed binary, or the self-contained
-  `rust-lld` linker.
+The flag accepts a comma-separated list of features, individually enabled (`+features`) or disabled
```

Comment:
I think that an example would be nice. It's a bit confusing that it says +features rather than +feature, and after reading this, I wouldn't be sure how to actually use it.

Reply:
Ah good catch, the `+features` is a typo from the unstable documentation. I've fixed it and added examples.